From 1eebfaff67aa6af7821fcc861fcdf3ef7fc9d25a Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 1 Sep 2016 12:54:08 -0400
Subject: [PATCH 48/49] Ticket 48957 - Update repl-monitor to handle new status
 messages

Bug Description:  The replication agreement status messages have changed,
                  and the repl-monitor script was not updated to reflect
                  these changes.  This lead to the html report incorrectly
                  color coding a successfull status.

Fix Description:  Update the script to ignore "Error (0)"

https://fedorahosted.org/389/ticket/48957

Reviewed by: mreynolds (one line commit rule)

(cherry picked from commit 32ee33b2222a9bbc0657ceb912ca3fa74ee27dcc)
(cherry picked from commit cf5683ae112528597af70a4e06cfb51e8e0c3c74)
---
 ldap/admin/src/scripts/repl-monitor.pl.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/admin/src/scripts/repl-monitor.pl.in b/ldap/admin/src/scripts/repl-monitor.pl.in
index aa7ab1e..a3efa8e 100755
--- a/ldap/admin/src/scripts/repl-monitor.pl.in
+++ b/ldap/admin/src/scripts/repl-monitor.pl.in
@@ -877,7 +877,7 @@ sub print_consumers
 			}   
 			my $redfontstart = "";
 			my $redfontend = "";
-			if ($status =~ /error/i) {
+			if ($status !~ /Error \(0\)/i) {
 			  $redfontstart = "<font color='red'>";
 			  $redfontend = "</font>";
 			}
-- 
2.4.11

